src/ir_Zepeal.cpp

```diff
@@ -0,0 +1,87 @@
+// Copyright 2020 Christian Nilsson (nikize)
+
+/// @file
+/// @brief Support for Zepeal protocol.
+/// This protocol uses fixed length bit encoding.
+/// Company behinde this brand seems to be Denkyosha
+/// @see https://www.denkyosha.co.jp/
+
+// Supports:
+//   Brand: Zepeal,  Model: DRT-A3311(BG) floor fan
+//   Brand: Zepeal,  Model: DRT-A3311(BG) 5 button remote
+
+#include "IRrecv.h"
+#include "IRsend.h"
+#include "IRutils.h"
+
+// Constants
+
+const uint16_t kZepealHdrMark = 2330;
+const uint16_t kZepealHdrSpace = 3380;
+const uint16_t kZepealOneMark = 1300;
+const uint16_t kZepealZeroMark = 420;
+const uint16_t kZepealOneSpace = kZepealZeroMark;
+const uint16_t kZepealZeroSpace = kZepealOneMark;
+const uint16_t kZepealFooterMark = 420;
+const uint16_t kZepealGap = 6750;
+
+const uint8_t  kZepealTolerance = 40;
+
+const uint8_t kZepealCheckExpected = 0x6C;
```

Comment:
Suggestion only: Signature or DataHeader may be better terms/description than CheckExpected. Completely optional/your call.